### DISCUSSION TITLE
Put some advertising on the site.

### DISCUSSION THREAD

**Discussion by Vagrin** [civitai/civitai#459] [Ideas]
There must be lots of companies that would love to advertise on your site. How many people are buying new video cards for SD, for instance? I bought a new NVidia card. Whole new computers. There are always some loud people that never want to pay for anything. They'll figure out how life works eventually. But advertising is the best way to pay for any service. And your site provides an amazing service extremely well. (When it's not seemingly overtaxed.) Don't go crazy. Just a strip down the side with advertisements for products that make generating AI art better and faster.

**Comment by JustMaier** [comment]
We like the idea of having sponsored spaces for contests that we plan to run in the future. We think it'd be awesome to work with Pre-built PC companies, GPU manufacturers, and even cloud compute companies to be able to award awesome prizes for the events. We've been planning something for a while, but haven't had the time to flesh out all the details.

So I guess, anyone reading this, if your interested in sponsoring a contest on Civitai, let me know...
